client/state/imports/actions.js

```diff
@@ -220,10 +228,4 @@ export const startUpload = ( importerStatus, file ) => dispatch => {
 		.then( fromApi )
```

Comment:
The `onabort` handler should be `() => dispatch( cancelImport( siteId, importerId ) )`